shared/build.gradle.kts

```diff
@@ -15,9 +15,10 @@ kotlin {
 		}
 		commonTest.dependencies {
 			implementation(libs.test.kotlin)
-			implementation(libs.test.kotest.runner)
+			implementation(libs.test.kotest.engine)
 			implementation(libs.test.mockk)
```

Comment:
mockk is a multiplatform library but it only supports a jvm or android source sets 
https://github.com/mockk/mockk/issues/950